Kinvey/Kinvey/FileStore.swift

```diff
@@ -219,7 +219,13 @@ open class FileStore {
                 }
                 
                 let handle: (Data?, URLResponse?, Swift.Error?) -> Void = { data, response, error in
-                    if let response = response as? HTTPURLResponse , 200 <= response.statusCode && response.statusCode < 300 {
+                    if self.client.logNetworkEnabled, let response = response as? HTTPURLResponse {
```

Comment:
It's up to you. If there's significant work, let's do it separately. I think this PR can be merged either way.